website/source/docs/builders/amazon-chroot.html.md

```diff
@@ -146,6 +146,8 @@ each category, the available configuration keys are alphabetized.
     process). Packer will always run this operation, even if the base
     AMI has an encrypted boot volume to start with. Default `false`.
 
+-   `insecure` (bool) - Explicitly allow the provider to perform "insecure" SSL requests. If omitted, default value is `false`.
```

Comment:
Please make sure the formatting is similar to other entries. ( return after ~80 characters )